cli/src/main/scala/com/lightbend/rp/reactivecli/annotations/Annotations.scala

```diff
@@ -73,7 +73,7 @@ case class Annotations(
  */
 object Annotations {
   def apply(labels: Map[String, String], args: GenerateDeploymentArgs): Annotations = {
-    val appVersion = version(labels)
+    val appVersion = version(labels, args)
```

Comment:
I think we should leave `version` as-is (have it only deal with labels), and add a `version: Option[String]` to `GenerateDeploymentArgs`.

This line (L76) then becomes `val appVersion = args.version.orElse(version(labels))`